mantidproject/mantid
# Serialise the Instrument View to Mantid project files

Saving a Mantid project file should now also include all of the necessary information for an instrument window to recreate itself afterwards

**To test:**
- Open the instrument view on a workspace with an instrument and play around with the different options. 
- Save the instrument view by saving to a Mantid project file.
- Close and reopen Mantid and reload the project. 
- The window should be recreated (along with any workspaces) exactly as it was before.

There are quite a lot of options on this interface so please try to be thorough and check as much as you can. One idea would be to run through the guide below and save the project periodically:

http://www.mantidproject.org/MantidPlot:_Instrument_View

Fixes #16962.

---
#### Reviewer

Please comment on the following ([full description](http://www.mantidproject.org/Individual_Ticket_Testing)):
##### Code Review
##### Functional Tests
- Does everything look good? Comment with the ship it emoji but don't merge. A member of `@mantidproject/gatekeepers` will take care of it.
